std/php7/Boot.hx

```diff
@@ -61,6 +61,13 @@ class Boot {
 	}
 
 	/**
+		Builds Haxe Array from array declaration syntax.
+	**/
+	public static function array( arr:NativeIndexedArray<Dynamic> ) : Array<Dynamic> {
```

Comment:
Even better. I can remove this method completely since in generated code we can directly call `Array.wrap()`. 
I just forgot about that when i was writing this method :)